tests/medium/test_web.py

```diff
@@ -65,17 +65,3 @@ def test_install_with_changed_download_page(self):
         umake_command = self.command('{} web phantomjs'.format(UMAKE))
         self.bad_download_page_test(umake_command, download_page_file_path)
         self.assertFalse(self.path_exists(self.exec_path))
-
-
-class VisualStudioCodeInContainer(ContainerTests, test_web.VisualStudioCodeTest):
```

Comment:
That should probably be in another PR? (the finale migation of vsc in IDE category)

Reply:
Yes..this was part of #442
It's fixed in my local branch...but it's not uploaded yet